test/position-managers/PositionManager.multicall.t.sol

```diff
@@ -0,0 +1,79 @@
+import {LiquidityAmounts} from "@uniswap/v4-core/test/utils/LiquidityAmounts.sol";
+import {TickMath} from "@uniswap/v4-core/src/libraries/TickMath.sol";
+import {TickMath} from "@uniswap/v4-core/src/libraries/TickMath.sol";
+import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
+import {Constants} from "@uniswap/v4-core/test/utils/Constants.sol";
+
+import {IERC20} from "forge-std/interfaces/IERC20.sol";
+import {ERC721} from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+
+import {IPositionManager, Actions} from "../../src/interfaces/IPositionManager.sol";
+import {PositionManager} from "../../src/PositionManager.sol";
+import {LiquidityRange, LiquidityRangeId, LiquidityRangeIdLibrary} from "../../src/types/LiquidityRange.sol";
+import {IMulticall} from "../../src/interfaces/IMulticall.sol";
+
+import {LiquidityFuzzers} from "../shared/fuzz/LiquidityFuzzers.sol";
+
+import {LiquidityOperations} from "../shared/LiquidityOperations.sol";
+import {Planner} from "../utils/Planner.sol";
+
+contract MulticallTest is Test, Deployers, GasSnapshot, LiquidityFuzzers, LiquidityOperations {
+    using FixedPointMathLib for uint256;
+    using CurrencyLibrary for Currency;
+    using LiquidityRangeIdLibrary for LiquidityRange;
+    using Planner for Planner.Plan;
+
+    PoolId poolId;
+    address alice = makeAddr("ALICE");
+
+    function setUp() public {
+        Deployers.deployFreshManagerAndRouters();
+        Deployers.deployMintAndApprove2Currencies();
+
+        (key, poolId) = initPool(currency0, currency1, IHooks(address(0)), 3000, SQRT_PRICE_1_1, ZERO_BYTES);
+
+        lpm = new PositionManager(manager);
+
+        IERC20(Currency.unwrap(currency0)).approve(address(lpm), type(uint256).max);
+        IERC20(Currency.unwrap(currency1)).approve(address(lpm), type(uint256).max);
+    }
+
+    function test_multicall_initializePool_mint() public {
+        key = PoolKey({currency0: currency0, currency1: currency1, fee: 0, tickSpacing: 10, hooks: IHooks(address(0))});
+
+        // Use multicall to initialize a pool and mint liquidity
+        bytes[] memory calls = new bytes[](2);
+        calls[0] = abi.encodeWithSelector(lpm.initializePool.selector, key, SQRT_PRICE_1_1, ZERO_BYTES);
+
+        LiquidityRange memory range = LiquidityRange({
+            poolKey: key,
+            tickLower: TickMath.minUsableTick(key.tickSpacing),
+            tickUpper: TickMath.maxUsableTick(key.tickSpacing)
+        });
+
+        Planner.Plan memory planner = Planner.init();
+        planner = planner.add(Actions.MINT, abi.encode(range, 100e18, address(this), ZERO_BYTES));
+        bytes memory actions = planner.finalize(range.poolKey);
+
+        calls[1] = abi.encodeWithSelector(IPositionManager.modifyLiquidities.selector, actions, _deadline);
+
+        IMulticall(address(lpm)).multicall(calls);
+
+        // test swap, doesn't revert
+        swap(key, true, -1e18, ZERO_BYTES);
```

Comment:
would be best to check the return values are non-0 too, to show that liquidity was added